File: feAPI/lib/models/lesson.dart
class Lesson {
  final int id;
  final int categoryId;
  final String title;
  final String description;

  Lesson({
    required this.id,
    required this.categoryId,
    required this.title,
    required this.description,
  });

  factory Lesson.fromJson(Map<String, dynamic> json) {
    return Lesson(
      id: json['id'],
      categoryId: json['category_id'],
      title: json['title'],
      description: json['description'],
    );
  }
}

File: backend_api/AppTiengAnhBE/Models/SystemModel/Lesson.cs
namespace AppTiengAnhBE.Domain.Entities
{
    public class Lesson
    {
        public int id { get; set; }
        public int category_id { get; set; }
        public string title { get; set; }
        public string description { get; set; }
        public int order_num { get; set; }
        public DateTime created_at { get; set; }
    }
}
